Remove meson (and python) dependency

jpcoles opened the pull request.

jpcoles-cscs committed:
.

jpcoles-cscs committed:
Remove meson (and python) dependency.

simonpintarelli commented on docker/Dockerfile:
```diff
@@ -31,25 +30,30 @@ RUN zypper install -y\
 
 # install meson and ninja via pip (packages are too old)
-RUN python3 -m pip install --upgrade pip && python3 -m pip install meson ninja
+#RUN python3 -m pip install --upgrade pip && python3 -m pip install meson ninja
```
Let us delete lines instead of commenting them.

jpcoles-cscs committed:
Remove commented lines in Dockerfile.
Fix SOURCE_DIR in {build,run}.sh scripts that needs to refer to the parent directory.
SOURCE_DIR is necessary to ensure the correct parent directory if the script is started from
another location, and --volume requires absolute paths.

jpcoles commented:
I agree about removing commented lines. I'm not used to people reading the code changes in a commit :)
I also made the SOURCE_DIR changes in this commit. It's still necessary to use SOURCE_DIR so that you a) are sure to have the correct parent if the script is started from another location and b) --volume requires an absolute path, so something like SOURCE_DIR would be necessary anyway.
